docs/src/pages/[platform]/components/message/react.mdx

```diff
@@ -0,0 +1,228 @@
+
+## Usage
+
+Message can be customized via props as a basic component, or it can be used as a <a href="#composable">composable component</a> which provides more control over layout and functionality.
+
+<Example>
+  <BasicUsage />
+  <ExampleCode>
+    ```jsx file=./examples/BasicUsage.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Color themes
+
+Message accepts the `colorTheme` prop which allows for different color variations. The available options are `neutral` (default), `error`, `info`, `success`, and `warning`. Messages that use a `error`, `info`, `success`, or `warning` colorTheme will show an icon by default.
+
+<Example>
+  <MessageColorThemes />
+  <ExampleCode>
+    ```jsx file=./examples/MessageColorThemes.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Icon
+
+The Message icon can be hidden by using the prop `hasIcon={false}`. See this section on <a href="#icons">using IconProvider to configure your own icons</a> for Message to use.
+
+<Example>
+  <HideIcon />
+  <ExampleCode>
+    ```jsx file=./examples/HideIcon.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Variations
+
+Message accepts the `variation` prop which allows for different background and border options.
+
+<Example>
+  <MessageVariations />
+  <ExampleCode>
+    ```jsx file=./examples/MessageVariations.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Dismissable
+
+Messages can be dismissed by setting `isDismissible={true}`. Additional dismiss behavior can be added by passing a function to the `onDismiss` prop. By default, we provide accessible button text that is hidden visually with <a href="/react/components/visuallyhidden">VisuallyHidden</a>; you can override this label by using the `dismissButtonLabel` prop.
+
+For more control over the appearance and functionality of the dismiss button, consider <a href="#composable">using Message as a composable component</a>.
+
+<Example>
+  <BasicWithDismiss />
+  <ExampleCode>
+    ```jsx file=./examples/BasicWithDismiss.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+## Composable
+
+For further control over the layout and child elements, Message includes composable child components to add extra layout customization: 
+- `Message.Container`
+- `Message.Content`
+- `Message.Heading`
+- `Message.Dismiss`
+- `Message.Icon`
+
+<Example>
+  <ComposableMessage />
+  <ExampleCode>
+    ```jsx file=./examples/ComposableMessage.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+### Customizing the Dismiss child component
+
+`Message.Dismiss` can be customized to hide the icon by setting the prop `hasIcon={false}`. It can also accept custom button text and other props that our <Link href="/react/components/button">Button</Link> primitive accepts.
+
+You can also override the dismiss functionality completely by passing an `onClick` event.
+
+<Example>
+  <WithCustomizedDismiss />
+  <ExampleCode>
+    ```jsx file=./examples/WithCustomizedDismiss.tsx
+
+    ```
+
+  </ExampleCode>
+</Example>
+
+## Styling
+
```

Comment:
I think you are missing the MessageThemeExample here

Reply:
Added that back in latest. Will have to update this and the theme/react links after merging to main.